### DISCUSSION TITLE
Πόσο "τουριστικό" μπορεί να είναι ένα συνέδριο μικτής πραγματικότητας;

### DISCUSSION THREAD

**Discussion by afoscolou** [sdy60/syllabus#49] [Q&A] [answered]
Θα ήθελα να ρωτήσω σχετικά με το concept του συνεδριακού τουρισμού και τι τελικά θα παραμείνει από αυτό αν εμείς οργανώσουμε, όπως μας ζητάει η εργασία, έναν ψηφιακό μετασχηματισμό του όπου οι σύνεδροι θα μπορούν να παρακολουθήσουν τα διάφορα events ασύγχρονα, υβριδικά, αλλά πάντως όχι με φυσική παρουσία.  Ο συνεδριακός τουρισμός, εκτός από τις συναντήσεις και τις συζητήσεις επί των θεμάτων της ατζέντας, περιλαμβάνει κι άλλες δραστηριότητες, όπως διαμονή σε ωραία ξενοδοχεία, γεύματα, περιηγήσεις, συμμετοχή σε εκδηλώσεις καθαρά για λόγους διασκέδασης των παρευρισκομένων.  Αναπτύσσοντας μια εφαρμογή για εμπειρία μικτής πραγματικότητας, ώστε οι σύνεδροι να μην είναι υποχρεωμένοι να βρίσκονται δια ζώσης στο συνέδριο, τους εξαιρούμε από τα περιφερειακά που προσδιορίζουν το "τουριστικό κομμάτι". Η ερώτησή μου είναι αν αυτό είναι αποδεκτό και αναμενόμενο στα πλαίσια της συγκεκριμένης εργασίας.

**Comment by ghost** [comment, marked as answer]
Μήπως το τουριστικό κομμάτι μεταφερθεί επίσης στη μικτή πραγματικότητα;

**Reply by epidrome** [reply]
σωστά, το νόημα-σκεπτικό του ερωτήματος είναι ότι κάποιοι θα έχουν φυσική παρουσία, το αν/τι/ποια τεχνολογία θα επαυξάνει την φυσική παρουσία είναι μέρος του σχεδιαστικού χώρου και των παραδοχών που πρέπει να γίνουν ώστε να εστιάσετε σε κάτι που θα κάνει την προσπάθεια σχεδίασης-υλοποίησης εφικτή.
